core/src/main/java/org/apache/accumulo/core/client/TServerStatus.java

```diff
@@ -0,0 +1,182 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.accumulo.core.client;
+
+/**
+ * This exception is thrown if a table is deleted after an operation starts.
+ *
+ * For example if table A exist when a scan is started, but is deleted during the scan then this exception is thrown.
+ *
+ */
+
+public class TServerStatus extends RuntimeException {
```

Comment:
I was following what I saw on the code in the other classes of the o.a.a.core.client package. Will remove it, thanks for pointing it out.